src/frontend/src/modules/Sidebar/SideBar.vue

```diff
@@ -191,7 +191,7 @@ export default {
 </script>
 <style>
 .sidebar {
-  background: #2b2b2b;
+  background: #2b2b2b !important;
```

Comment:
This is a great solution! 

Just one small suggestion—have you thought about addressing the root cause in `/src/frontend/src/assets/sass/mpm/_responsive.scss`? It looks like the `background-color: #9a9a9a` there might be the source of the issue. Fixing it directly at the source could make the solution even cleaner. Let me know what you think!

Reply:
@beesaferoot makes sense fixed  you can check again